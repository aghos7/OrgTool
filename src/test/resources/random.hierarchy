1, null, Org1
2, 1, Org2
8, 1, Org3
6, 2, Org6
11, 10, Org11
10, 3, Org10
23, null, Org23
7, 6, Org7
12, 6, Org8
9, 8, Org9