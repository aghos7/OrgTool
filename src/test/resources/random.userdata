1, 1, 10, 20
2, 2, 15, 5
3, 3, 2, 1