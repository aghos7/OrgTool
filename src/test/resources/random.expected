null, 3, 27, 26
   1, 2, 25, 25
      2, 1, 15, 5
         6, 0, 0, 0
            7, 0, 0, 0
            12, 0, 0, 0
      8, 0, 0, 0
         9, 0, 0, 0
   3, 1, 2, 1
      10, 0, 0, 0
         11, 0, 0, 0
   23, 0, 0, 0